staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -104,11 +105,23 @@ func (s *ServerRunOptions) Validate() []error {
 	if s.TargetRAMMB < 0 {
 		errors = append(errors, fmt.Errorf("--target-ram-mb can not be negative value"))
 	}
-	if s.MaxRequestsInFlight < 0 {
-		errors = append(errors, fmt.Errorf("--max-requests-inflight can not be negative value"))
-	}
-	if s.MaxMutatingRequestsInFlight < 0 {
-		errors = append(errors, fmt.Errorf("--max-mutating-requests-inflight can not be negative value"))
+
+	if s.EnableInfightQuotaHandler {
+		if s.MaxMutatingRequestsInFlight != 0 {
```

Comment:
Ideally we'd be checking if the user specified the flag or not, it's a bit weird to make them explicitly set the flag to 0 :/

Reply:
mmm.. yeah ideally.. while the cli framework doesn't seem support that yet.